modules/loader-utils/src/worker-utils/create-worker.js

```diff
@@ -88,7 +91,7 @@ async function parseData({loader, arraybuffer, byteOffset, byteLength, options,
 
 // Filter out noise messages sent to workers
 function isKnownMessage(data, name) {
-  return data && data.type === 'parse' && data.source === 'loaders.gl';
+  return data && data.type === 'parse' && data.source && data.source.startsWith('loaders.gl');
```

Comment:
Nit: Do we even need to check for `parse` in this case?

Reply:
Yes, the worker can also receive message from its previously submitted parse requests to sub workers.